version: Show the agent version in startup

jcvenegas opened the pull request:
This patch adds a message to show the agent version
at startup, this will be useful for debugging.

Fixes: #52

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

jcvenegas commented:
@sboeuf  what do you think about save a commit in 'commit_id' file and use it, this  in order to show the commit id from a *make dist* tarball (that we dont have yet)
Edit: Added issue for `make dist` https://github.com/clearcontainers/agent/issues/54

sboeuf commented on agent.go:
```diff
@@ -159,7 +159,11 @@ func init() {
 	}
 }
 
+// Version is the proxy version. This variable is populated at build time.
```
s/proxy/agent/

sboeuf commented:
@jcvenegas yes if you want to display the commit id on the tarball name, I am fine with this :)

jcvenegas committed:
version: Show the agent version in startup

This patch adds a message to show the agent version
at startup, this will be useful for debugging.

Fixes: #52

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

jodh-intel commented:
Great! This is indeed going to be extremely useful.

lgtm

[![Approved with PullApprove](https://img.shields.io/badge/reviewers-approved-brightgreen.svg)](https://pullapprove.com/clearcontainers/agent/pull-request/53/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=clearcontainers/agent)
